src/registrar/tests/test_views.py

```diff
@@ -1406,6 +1406,40 @@ def test_domain_your_contact_information_content(self):
         )
         self.assertContains(page, "Testy")
 
+    def test_domain_security_email_existing_security_contact(self):
+        """Can load domain's security email page."""
+        self.mockSendPatch = patch("registrar.models.domain.registry.send")
```

Comment:
Test interference reasons as the parent doesn't import MockEppLib (I believe). I think I did this initially and it would have required a refactor of multiple tests so this minimizes code interference